guides/v2.1/cloud/release-notes/cloud-tools.md

```diff
@@ -57,7 +64,7 @@ Cloud configuration:
 
 ## v2002.0.11
 
-{% include note.html type="info" content="The Cloud tools version 2002.0.11 now supports Magento 2.1.13. " %}
+{% include note.html type="info" content="The ece-tools version 2002.0.11 now supports Magento 2.1.13. " %}
```

Comment:
I think we should use inline attributes to define notes rather than liquid + includes because the latter doesn't support certain things. For example, I don't think you can use Jekyll variables when you use this format. This could lead to broken things in the future. See [Style Guide](https://wiki.corp.magento.com/display/MDOC/Style+guide+-+formatting#Styleguide-formatting-Links).

Preferred format:

```
{:.bs-callout bs-callout-info}
This is a note.
```